extensions/data-transfer/portability-data-transfer-instagram/src/main/java/org/dataportabilityproject/transfer/instagram/photos/InstagramPhotoImporter.java

```diff
@@ -31,6 +31,7 @@
   @Override
   public ImportResult importItem(UUID jobId, TokensAndUrlAuthData authData,
       PhotosContainerResource data) {
-    return null; // TODO: implement
+    // TODO(#337): Import is not supported for Instagram since their API does not support upload
```

Comment:
tagging #337  explicitly so it ties the PR to the issue

Reply:
Can we just delete the importer and throw an error in the TransferServiceExtension?

Reply:
If you don't mind, I'll leave this up to the implementation of #337 - I think we'll probably end up doing it the way you described but in case we end up doing something else.

Reply:
sgtm!